GameServer/Logic/Maps/SummonersRift.cs

```diff
@@ -106,49 +108,49 @@ public SummonersRift(Game game) : base(game, 90 * 1000, 30 * 1000, 90 * 1000, tr
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_T1_C_02_A", 1768.19f, 1589.47f, TeamId.TEAM_BLUE, TurretType.NexusTurret));
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_T2_C_05_A", 8548.0f, 8289.0f, TeamId.TEAM_PURPLE, TurretType.OuterTurret));
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_T2_C_04_A", 9361.0f, 9892.0f, TeamId.TEAM_PURPLE, TurretType.InnerTurret));
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_T2_C_03_A", 10743.0f, 11010.0f, TeamId.TEAM_PURPLE, TurretType.InhibitorTurret));
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_T2_C_01_A", 12662.0f, 12442.0f, TeamId.TEAM_PURPLE, TurretType.NexusTurret));
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_T2_C_02_A", 12118.0f, 12876.0f, TeamId.TEAM_PURPLE, TurretType.NexusTurret));
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_OrderTurretShrine_A", -236.05f, -53.32f,TeamId.TEAM_BLUE, TurretType.FountainTurret));
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_ChaosTurretShrine_A", 14157.0f, 14456.0f, TeamId.TEAM_PURPLE, TurretType.FountainTurret));
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_T1_L_03_A", 574.66f, 10220.47f, TeamId.TEAM_BLUE, TurretType.OuterTurret));
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_T1_L_02_A", 1106.26f, 6485.25f, TeamId.TEAM_BLUE, TurretType.InnerTurret));
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_T1_C_06_A", 802.81f, 4052.36f, TeamId.TEAM_BLUE, TurretType.InhibitorTurret));
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_T2_L_03_A", 3911.0f, 13654.0f, TeamId.TEAM_PURPLE, TurretType.OuterTurret));
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_T2_L_02_A", 7536.0f, 13190.0f, TeamId.TEAM_PURPLE, TurretType.InnerTurret));
-            AddObject(new LaneTurret(game, game.GetNewNetID(), "Turret_T2_L_01_A", 10261.0f, 13465.0f, TeamId.TEAM_PURPLE, TurretType.InhibitorTurret));
-
-            AddObject(new LevelProp(game, game.GetNewNetID(), 12465.0f, 14422.257f, 101.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, "LevelProp_Yonkey", "Yonkey"));
-            AddObject(new LevelProp(game, game.GetNewNetID(), -76.0f, 1769.1589f, 94.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, "LevelProp_Yonkey1", "Yonkey"));
-            AddObject(new LevelProp(game, game.GetNewNetID(), 13374.17f, 14245.673f, 194.9741f, 224.0f, 33.33f, 0.0f, 0.0f, -44.44f, "LevelProp_ShopMale", "ShopMale"));
-            AddObject(new LevelProp(game, game.GetNewNetID(), -99.5613f, 855.6632f, 191.4039f, 158.0f, 0.0f, 0.0f, 0.0f, 0.0f, "LevelProp_ShopMale1", "ShopMale"));
+            AddObject(new LaneTurret(_networkIdManager.GetNewNetID(), "Turret_T1_R_03_A", 10097.62f, 808.73f, TeamId.TEAM_BLUE, TurretType.OuterTurret));
```

Comment:
120 character limit for all of these, but I don't think you should bother changing this.